pkg/api/types.go

```diff
@@ -901,13 +901,14 @@ type NamespaceList struct {
 	Items []Namespace `json:"items"`
 }
 
-// Binding is written by a scheduler to cause a pod to be bound to a host.
+// Binding ties one object to another - for example, a pod is bound to a node by a scheduler.
 type Binding struct {
-	TypeMeta   `json:",inline"`
+	TypeMeta `json:",inline"`
+	// ObjectMeta describes the object that is being bound.
 	ObjectMeta `json:"metadata,omitempty"`
 
-	PodID string `json:"podID"`
-	Host  string `json:"host"`
+	// To is the object to bind to.
+	To ObjectReference `json:"to" description:"to is the object to bind to"`
```

Comment:
Maybe `Target` rather than `To`? I had to read this line twice to parse it.

Also, this is the internal object. Do we want descriptions on these fields? We don't have any so far, and we plan to eliminate the json tags eventually.